tests/integration/suite/scheduler/api/errors.go

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+import (
+	"context"
+	stderrors "errors"
+	"testing"
+
+	"github.com/dapr/dapr/tests/integration/framework/process/grpc/scheduler"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/genproto/googleapis/rpc/errdetails"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	apierrors "github.com/dapr/dapr/pkg/api/errors"
+	rtv1 "github.com/dapr/dapr/pkg/proto/runtime/v1"
+	schedulerv1pb "github.com/dapr/dapr/pkg/proto/scheduler/v1"
+	"github.com/dapr/dapr/tests/integration/framework"
+	"github.com/dapr/dapr/tests/integration/framework/process/daprd"
+	"github.com/dapr/dapr/tests/integration/framework/process/exec"
+	"github.com/dapr/dapr/tests/integration/framework/process/sentry"
+	"github.com/dapr/dapr/tests/integration/suite"
+)
+
+func init() {
+	suite.Register(new(errors))
+}
+
+type errors struct {
+	daprd     *daprd.Daprd
+	sentry    *sentry.Sentry
```

Comment:
It looks like we want to swap this test case with the ones that live in `tests/integration/suite/daprd/scheduler/http/errors.go` and `tests/integration/suite/daprd/scheduler/grpc/errors.go`, as the one here is not actually running `scheduler` where the ones there are.